masim.h

```diff
@@ -16,18 +18,18 @@ enum rw_mode {
 struct access {
 	struct mregion *mregion;
 	int random_access;
-	size_t stride;
+	uint64_t stride;
 	int probability;
 	enum rw_mode rw_mode;
 
 	/* For runtime only */
 	int prob_start;
-	size_t last_offset;
+	uint64_t last_offset;
 };
 
 struct phase {
 	char *name;
-	unsigned time_ms;
+	uint64_t time_ms;
```

Comment:
Questions about `size_t` size of 64bit machines and 64bit support of time applies to above changes, too.